documentation/manual/README.md

```diff
@@ -6,7 +6,7 @@ Background
 
 The OSHDB was initiated as a project to make it easier for researchers to investigate data quality aspects of the OpenStreetMap (OSM) data though analysing its history.
 
-OSM is a rich resource of freely available geographic information. Analysing historical OSM data provides great insight into the evolution of the map. This supports assessing OSM data quality intrinsically, i.e., without the need for external reference data sets. However, the possibilities for analyzing the OSM history data on a global scale are limited because of the large amount of resources needed and the lack of easy to use analytics software. That is, OSM’s huge information treasure for researchers, journalists, community members and other interested people is kept hidden.
+OSM is a rich resource of freely available geographic information. Analysing historical OSM data provides great insight into the evolution of the map. This evolution supports assessing OSM data quality intrinsically, i.e., without the need for external reference data sets. However, the possibilities for analysing the OSM history data on a global scale are limited because of the large amount of resources needed, and the lack of easy to use analytics software. That is, OSM’s huge information treasure for researchers, journalists, community members and other interested people is kept hidden.
```

Comment:
see comment about the usage of a serial comma above.

Reply:
✔